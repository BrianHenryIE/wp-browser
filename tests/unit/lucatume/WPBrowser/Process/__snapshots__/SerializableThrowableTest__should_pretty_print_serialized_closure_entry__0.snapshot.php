array (
  0 => 
  array (
    'file' => 'closure://static function () {
   1|            $set = \'foo\';
   2|            $bar = \'bar\';
   3|
 > 4|            self::throwingMethod();
   5|        }',
    'line' => 4,
    'function' => 'throwingMethod',
    'class' => 'lucatume\\WPBrowser\\Process\\SerializableThrowableTest',
    'type' => '::',
  ),
  1 => 
  array (
    'function' => '{closure}',
    'class' => 'lucatume\\WPBrowser\\Process\\SerializableThrowableTest',
    'type' => '::',
  ),
  2 => 
  array (
    'file' => '/vendor/opis/closure/src/SerializableClosure.php',
    'line' => 109,
    'function' => 'call_user_func_array',
  ),
  3 => 
  array (
    'file' => '/tests/unit/lucatume/WPBrowser/Process/SerializableThrowableTest.php',
    'line' => 58,
    'function' => '__invoke',
    'class' => 'Opis\\Closure\\SerializableClosure',
    'type' => '->',
  ),
  4 => 
  array (
    'file' => '/vendor/phpunit/phpunit/src/Framework/TestCase.php',
    'line' => 1608,
    'function' => 'should_pretty_print_serialized_closure_entry',
    'class' => 'lucatume\\WPBrowser\\Process\\SerializableThrowableTest',
    'type' => '->',
  ),
  5 => 
  array (
    'file' => '/vendor/phpunit/phpunit/src/Framework/TestCase.php',
    'line' => 1214,
    'function' => 'runTest',
    'class' => 'PHPUnit\\Framework\\TestCase',
    'type' => '->',
  ),
  6 => 
  array (
    'file' => '/vendor/codeception/codeception/src/Codeception/Test/TestCaseWrapper.php',
    'line' => 148,
    'function' => 'runBare',
    'class' => 'PHPUnit\\Framework\\TestCase',
    'type' => '->',
  ),
  7 => 
  array (
    'file' => '/vendor/codeception/codeception/src/Codeception/Test/Test.php',
    'line' => 170,
    'function' => 'test',
    'class' => 'Codeception\\Test\\TestCaseWrapper',
    'type' => '->',
  ),
  8 => 
  array (
    'file' => '/vendor/codeception/codeception/src/Codeception/Suite.php',
    'line' => 130,
    'function' => 'realRun',
    'class' => 'Codeception\\Test\\Test',
    'type' => '->',
  ),
  9 => 
  array (
    'file' => '/vendor/codeception/codeception/src/Codeception/SuiteManager.php',
    'line' => 148,
    'function' => 'run',
    'class' => 'Codeception\\Suite',
    'type' => '->',
  ),
  10 => 
  array (
    'file' => '/vendor/codeception/codeception/src/Codeception/Codecept.php',
    'line' => 260,
    'function' => 'run',
    'class' => 'Codeception\\SuiteManager',
    'type' => '->',
  ),
  11 => 
  array (
    'file' => '/vendor/codeception/codeception/src/Codeception/Codecept.php',
    'line' => 216,
    'function' => 'runSuite',
    'class' => 'Codeception\\Codecept',
    'type' => '->',
  ),
  12 => 
  array (
    'file' => '/vendor/codeception/codeception/src/Codeception/Command/Run.php',
    'line' => 435,
    'function' => 'run',
    'class' => 'Codeception\\Codecept',
    'type' => '->',
  ),
  13 => 
  array (
    'file' => '/vendor/symfony/console/Command/Command.php',
    'line' => 298,
    'function' => 'execute',
    'class' => 'Codeception\\Command\\Run',
    'type' => '->',
  ),
  14 => 
  array (
    'file' => '/vendor/symfony/console/Application.php',
    'line' => 1040,
    'function' => 'run',
    'class' => 'Symfony\\Component\\Console\\Command\\Command',
    'type' => '->',
  ),
  15 => 
  array (
    'file' => '/vendor/symfony/console/Application.php',
    'line' => 301,
    'function' => 'doRunCommand',
    'class' => 'Symfony\\Component\\Console\\Application',
    'type' => '->',
  ),
  16 => 
  array (
    'file' => '/vendor/symfony/console/Application.php',
    'line' => 171,
    'function' => 'doRun',
    'class' => 'Symfony\\Component\\Console\\Application',
    'type' => '->',
  ),
  17 => 
  array (
    'file' => '/vendor/codeception/codeception/src/Codeception/Application.php',
    'line' => 112,
    'function' => 'run',
    'class' => 'Symfony\\Component\\Console\\Application',
    'type' => '->',
  ),
  18 => 
  array (
    'file' => '/vendor/codeception/codeception/app.php',
    'line' => 45,
    'function' => 'run',
    'class' => 'Codeception\\Application',
    'type' => '->',
  ),
  19 => 
  array (
    'function' => '{closure}',
  ),
  20 => 
  array (
    'file' => '/vendor/codeception/codeception/app.php',
    'line' => 46,
    'function' => 'call_user_func',
  ),
  21 => 
  array (
    'file' => '/vendor/codeception/codeception/codecept',
    'line' => 7,
    'function' => 'require',
  ),
  22 => 
  array (
    'file' => '/vendor/bin/codecept',
    'line' => 120,
    'function' => 'include',
  ),
)