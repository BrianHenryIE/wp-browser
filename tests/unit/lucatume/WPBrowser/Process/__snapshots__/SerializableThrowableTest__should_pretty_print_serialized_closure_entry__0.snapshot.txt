#0 closure://static function () {
   1|            $set = 'foo';
   2|            $bar = 'bar';
   3|
[35m > 4|            self::throwingMethod();[0m
   5|        }(4): lucatume\WPBrowser\Process\SerializableThrowableTest::throwingMethod()
#1 [internal function]: lucatume\WPBrowser\Process\SerializableThrowableTest::{closure}()
#2 /vendor/opis/closure/src/SerializableClosure.php(109): call_user_func_array()
#3 /tests/unit/lucatume/WPBrowser/Process/SerializableThrowableTest.php(58): Opis\Closure\SerializableClosure->__invoke()
#4 /vendor/phpunit/phpunit/src/Framework/TestCase.php(1608): lucatume\WPBrowser\Process\SerializableThrowableTest->should_pretty_print_serialized_closure_entry()
#5 /vendor/phpunit/phpunit/src/Framework/TestCase.php(1214): PHPUnit\Framework\TestCase->runTest()
#6 /vendor/codeception/codeception/src/Codeception/Test/TestCaseWrapper.php(148): PHPUnit\Framework\TestCase->runBare()
#7 /vendor/codeception/codeception/src/Codeception/Test/Test.php(170): Codeception\Test\TestCaseWrapper->test()
#8 /vendor/codeception/codeception/src/Codeception/Suite.php(130): Codeception\Test\Test->realRun()
#9 /vendor/codeception/codeception/src/Codeception/SuiteManager.php(148): Codeception\Suite->run()
#10 /vendor/codeception/codeception/src/Codeception/Codecept.php(260): Codeception\SuiteManager->run()
#11 /vendor/codeception/codeception/src/Codeception/Codecept.php(216): Codeception\Codecept->runSuite()
#12 /vendor/codeception/codeception/src/Codeception/Command/Run.php(646): Codeception\Codecept->run()
#13 /vendor/codeception/codeception/src/Codeception/Command/Run.php(467): Codeception\Command\Run->runSuites()
#14 /vendor/symfony/console/Command/Command.php(298): Codeception\Command\Run->execute()
#15 /vendor/symfony/console/Application.php(1040): Symfony\Component\Console\Command\Command->run()
#16 /vendor/symfony/console/Application.php(301): Symfony\Component\Console\Application->doRunCommand()
#17 /vendor/symfony/console/Application.php(171): Symfony\Component\Console\Application->doRun()
#18 /vendor/codeception/codeception/src/Codeception/Application.php(112): Symfony\Component\Console\Application->run()
#19 /vendor/codeception/codeception/app.php(45): Codeception\Application->run()
#20 [internal function]: {closure}()
#21 /vendor/codeception/codeception/app.php(46): call_user_func()
#22 /vendor/codeception/codeception/codecept(7): require()
#23 /vendor/bin/codecept(120): include()
#24 {main}